Source/RxSwift/nwise.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  nwise.swift
+//  RxSwiftExt
+//
+//  Created by Zsolt Váradi on 2017. 12. 09..
+//  Copyright © 2017. RxSwiftCommunity. All rights reserved.
+//
+
+import RxSwift
+
+extension ObservableType {
+    /**
+     Groups the elements of the source observable into arrays of N consecutive elements.
+
+     The resulting observable:
+     - does not emit anything until the source emits at least N elements;
+     - emits an array for every element after that;
+     - and forwards any error or completed event.
+
+     For example:
+
+          --(1)--(2)--(3)-------(4)-------(5)------->
+           |
+           | nwise(3)
+           v
+          ------------([1,2,3])-([2,3,4])-([3,4,5])->
```

Comment:
👏 👏 👏 👏 👏  Fan of ASCII diagrams !

Reply:
Thanks, me too :) And I was pleasantly surprised how well it looks in Xcode's quick help bubbles.